tests/index.spec.ts

```diff
@@ -35,6 +35,11 @@ describe("addFormats options", () => {
     expect(validateTime("17:27:38Z")).toEqual(true)
     expect(validateTime("25:27:38Z")).toEqual(true)
     expect(validateTime("17:27")).toEqual(false)
+
+    const validateDatetime = ajv.compile({format: "date-time"})
+    expect(validateDatetime("2016-12-31T23:59:60Z")).toEqual(true)
+    expect(validateDatetime("2016-15-31T23:59:60Z")).toEqual(true)
```

Comment:
This should be false.
```suggestion
    expect(validateDatetime("2016-15-31T23:59:60Z")).toEqual(false)